mlflow/utils/rest_utils.py

```diff
@@ -14,22 +13,14 @@ def get_databricks_hostname_and_auth():
     by the Databricks CLI. Returns a tuple of (hostname, auth, token) to use when making API
     requests.
     """
-    home_dir = os.path.expanduser("~")
-    cfg_file = os.path.join(home_dir, ".databrickscfg")
-    if not os.path.exists(cfg_file):
-        raise Exception("Could not find profile for Databricks CLI in %s. Make sure the the "
-                        "Databricks CLI is installed and that credentials have been "
-                        "configured as described in "
-                        "https://github.com/databricks/databricks-cli" % cfg_file)
-    else:
-        config = provider.get_config_for_profile(provider.DEFAULT_SECTION)
-        if config.username is not None and config.password is not None:
-            return config.host, (config.username, config.password), config.token
-        return config.host, None, config.token
-
+    config = provider.get_config_for_profile(provider.DEFAULT_SECTION)
+    if config.username is not None and config.password is not None:
+        return config.host, (config.username, config.password), config.token
+    return config.host, None, config.token
```

Comment:
Maybe we can defensively say `if config.token is not None` (or just `if config.token:`) and in the else condition throw an appropriate error.